test_trained_model.py

```diff
@@ -26,66 +26,76 @@ def test_trained_model(config_file: str = None, chosen_model: torch.nn.Module =
     config = {}
     with open(config_file, "r") as f:
         config = json.load(f)
-    predicted_value_option = {1: 1, 2: 32, 3: 32, 4: 33, 5: 33, 6: 65}
-    config["output_dim"] = predicted_value_option[config["predicted_value_option"]]
-
-    dataset_options = {
-        1: Dataset.CuAu,
-        2: Dataset.FePt,
-        3: Dataset.CuAu_FePt_SHUFFLE,
-        4: Dataset.CuAu_TRAIN_FePt_TEST,
-        5: Dataset.FePt_TRAIN_CuAu_TEST,
-        6: Dataset.FeSi,
-        7: Dataset.FePt_FeSi_SHUFFLE,
-        8: Dataset.unit_test,
-    }
-    chosen_dataset_option = None
-    for dataset in dataset_options.values():
-        if dataset.value == config["dataset_option"]:
-            chosen_dataset_option = dataset
 
     train_loader, val_loader, test_loader = dataset_loading_and_splitting(
         config=config,
-        chosen_dataset_option=chosen_dataset_option,
+        chosen_dataset_option=config["Dataset"]["name"],
```

Comment:
I thought that test_trained_model would disappear. 
@streeve @pzhanggit  why is it still here?

Reply:
You call it inside `test_random_graphs`

Reply:
Are there any reasons that make us want it disappear? The current implementation looks good to me. 
The are some details we can improve after moving to ORNL.